### ISSUE TITLE
SpikeSortingError: Spike sorting failed: klusta returned a non-zero exit code.

### ISSUE THREAD

**Issue by AlanoudAlsubhi** [SpikeInterface/spikesorters#211]
Hi again, 

I am running SpikeInterface with the Klusta sorter on MacOS Catalina. It starts the analysis but then it stops with an error. I did run the same code on my window10 and it works fine. 
<img width="830" alt="Screen Shot 2021-03-25 at 10 24 06 AM" src="https://user-images.githubusercontent.com/79871345/112488805-42a7e200-8d54-11eb-97ce-d4a6c91aa9a9.png">
<img width="773" alt="Screen Shot 2021-03-25 at 10 23 11 AM" src="https://user-images.githubusercontent.com/79871345/112488831-476c9600-8d54-11eb-9c33-bc01ed82c6f3.png">

Thank you so much for your help.

**Comment by alejoe91** [comment]
Hi,

Can you check the klusta.log in the output folders? Not sure why it's failing

**Comment by alejoe91** [comment]
It could be that it found no units on a tetrode. Can you try to run it with the `raise_error=False` argument?

**Comment by AlanoudAlsubhi** [comment]
[klusta.log](https://github.com/SpikeInterface/spikesorters/files/6205449/klusta.log)

ValueError: cannot convert float NaN to integer

**Comment by alejoe91** [comment]
Yeah that means spike sorting failed for some reason..can you try what I suggested? Maybe that happens only for one tetrode

**Comment by AlanoudAlsubhi** [comment]
So I did add raise_error=False as an argument. 

ValueError: cannot convert float NaN to integer

Error running klusta
No spikes found for this channel group
Klusta found 0 units. 

Does it stop if doesn't find any units on one of the channels on the first group?

**Comment by alejoe91** [comment]
It might unfortunately. I would refer you to the klusta documentation, but klusta is.not being developed anymore

**Comment by AlanoudAlsubhi** [comment]
I see. I found it odd that it found units when I ran it on my window10 but not on Mac. 
do you recommend another sorter that runs well on both mac and windows10 and doesn't require Matlab? 

Thanks
